rasa/core/agent.py

```diff
@@ -400,6 +400,30 @@ def is_ready(self):
             and self.interpreter is not None
         )
 
+    async def parse_message_from_text(self, message_data: Text):
+        """Handles message text and intent payload input messages.
```

Comment:
I don't get this one. how about `Parse message using NLU interpreter` ?

Reply:
please add the return type as we discussed in our call :-)

Reply:
Done